tests/lean/interactive/completionOption.lean

```diff
@@ -0,0 +1,2 @@
+set_option pp
+           --^ textDocument/completion
```

Comment:
Btw, please make sure to end files with a trailing newline to avoid unnecessary diffs. Can we commit a `.vscode/` that does that?